src/components/screens/monitor/accounts/accounts.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import MonitorHeader from '../header';
+import AccountsTable from '../../../shared/accountsTable';
```

Comment:
```suggestion
import AccountsTable from './accountsTable';
```
💡 `AccountsTable` is not really `shared`, as it's used only here and AFAIK there is no plan to use it from somewhere else. I think you copied this pattern from one of these:
- `TransactionsTable`, which is used in both `/monitor/transactions` and `/monitor/blocks/<BLOCK_ID> `
- `DelegatesTable`, which is used in both `/delegates` and `/monitor/delegates/`

In the end, it's not a big deal, so I don't insist on moving it, just wanted to mention it.